app/controllers/api/items_controller.rb

```diff
@@ -0,0 +1,74 @@
+module API
+  class ItemsController < ApplicationController
```

Comment:
This class seems specific to Arkivo, but "ItemsController" doesn't tell me much. How about adding some documentation about what this class is for.